torch/nested/_internal/ops.py

```diff
@@ -36,7 +36,8 @@ def check_schema(schema_str: str, func, *args, **kwargs) -> None:
 
     arg_type_check_fns = {
         "t": lambda x: isinstance(x, torch.Tensor) and not isinstance(x, NestedTensor),
-        "jt": lambda x: isinstance(x, NestedTensor),
+        "jt": lambda x: isinstance(x, NestedTensor) and x._lengths is None,
```

Comment:
nit: maybe add a small comment describing that `jt` requires contiguous JTs only